pkg/app/postgresql.go

```diff
@@ -3,79 +3,73 @@ package app
 )
 
-const (
-	PostgresHostFlag    = "postgres-host"
-	DefaultPostgresHost = "127.0.0.1"
-
-	PostgresPortFlag    = "postgres-port"
-	DefaultPostgresPort = 5432
-
-	PostgresUserFlag    = "postgres-user"
-	DefaultPostgresUser = "go_project_template"
-
-	PostgresPasswordFlag    = "postgres-password"
-	DefaultPostgresPassword = "go_project_template"
-
-	PostgresDatabaseFlag = "postgres-database"
-
-	PostgresMigrationPath = "migration-path"
+var (
+	PostgresHost = cli.StringFlag{ // nolint: gochecknoglobals
+		Name:   "postgres-host",
+		Usage:  "PostgresSQL host to connect",
+		EnvVar: "POSTGRES_HOST",
+		Value:  "127.0.0.1",
+	}
+	PostgresPort = cli.IntFlag{ // nolint: gochecknoglobals
+		Name:   "postgres-port",
+		Usage:  "PostgresSQL port to connect",
+		EnvVar: "POSTGRES_PORT",
+		Value:  5432, // nolint: gomnd
+	}
+	PostgresUser = cli.StringFlag{ // nolint: gochecknoglobals
+		Name:   "postgres-user",
+		Usage:  "PostgresSQL user to connect",
+		EnvVar: "POSTGRES_USER",
+		Value:  "go_project_template",
+	}
+	PostgresPassword = cli.StringFlag{ // nolint: gochecknoglobals
+		Name:   "postgres-password",
+		Usage:  "PostgresSQL password to connect",
+		EnvVar: "POSTGRES_PASSWORD",
+		Value:  "go_project_template",
+	}
+	PostgresDatabase = cli.StringFlag{ // nolint: gochecknoglobals
+		Name:   "postgres-database",
+		Usage:  "Postgres database to connect",
+		EnvVar: "POSTGRES_DATABASE",
+		Value:  "go_project_template",
+	}
+	PostgresMigrationPath = cli.StringFlag{ // nolint: gochecknoglobals
+		Name:   "migration-path",
+		Value:  "migrations",
+		EnvVar: "MIGRATION_PATH",
+	}
 )
 
 // PostgresSQLFlags creates new cli flags for PostgreSQL client.
 func PostgresSQLFlags(defaultDB string) []cli.Flag {
+	db := PostgresDatabase
+	db.Value = defaultDB
+
 	return []cli.Flag{
-		cli.StringFlag{
-			Name:   PostgresHostFlag,
-			Usage:  "PostgresSQL host to connect",
-			EnvVar: "POSTGRES_HOST",
-			Value:  DefaultPostgresHost,
-		},
-		cli.IntFlag{
-			Name:   PostgresPortFlag,
-			Usage:  "PostgresSQL port to connect",
-			EnvVar: "POSTGRES_PORT",
-			Value:  DefaultPostgresPort,
-		},
-		cli.StringFlag{
-			Name:   PostgresUserFlag,
-			Usage:  "PostgresSQL user to connect",
-			EnvVar: "POSTGRES_USER",
-			Value:  DefaultPostgresUser,
-		},
-		cli.StringFlag{
-			Name:   PostgresPasswordFlag,
-			Usage:  "PostgresSQL password to connect",
-			EnvVar: "POSTGRES_PASSWORD",
-			Value:  DefaultPostgresPassword,
-		},
-		cli.StringFlag{
-			Name:   PostgresDatabaseFlag,
-			Usage:  "Postgres database to connect",
-			EnvVar: "POSTGRES_DATABASE",
-			Value:  defaultDB,
-		},
-		cli.StringFlag{
-			Name:   PostgresMigrationPath,
-			Value:  "migrations",
-			EnvVar: "MIGRATION_PATH",
-		},
+		PostgresHost,
+		PostgresPort,
+		PostgresUser,
+		PostgresPassword,
+		db,
+		PostgresMigrationPath,
 	}
 }
 
 // NewDBFromContext creates a DB instance from cli flags configuration.
 func NewDBFromContext(c *cli.Context) (*sqlx.DB, error) {
```

Comment:
I think we should add `NewDB` that will take a config for setting up database and then `NewDBFromContext` can call it.

Reply:
fixed